README.md

```diff
@@ -1,2 +1,86 @@
-# Fire-Extinguisher-Bot
-Open Projects 2021
+# **Fire-Extinguisher-Bot**
```

Comment:
Give space after open projects.

Reply:
do i hv to give a new line after "open projects 2021"??